airbyte-integrations/connectors/source-bing-ads/unit_tests/integrations/config.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+
+import datetime
+from typing import Any, Dict, List
+
+from airbyte_cdk.test.mock_http.response_builder import find_template
+
+TENNANT_ID = "common"
+DEVELOPER_TOKEN = "test-token"
+REFRESH_TOKEN = "test-refresh-token"
+CLIENT_ID = "test-client-id"
+CLIENT_SECRET = "test-client-secret"
+LOOKBACK_WINDOW = 0
+
+
+class ConfigBuilder:
+    def __init__(self) -> None:
+        oauth_fixture: Dict[str, Any] = find_template("oauth", __file__)
+        self._access_token: str = oauth_fixture["access_token"]
+        self._refresh_token: str = oauth_fixture["refresh_token"]
+        self._client_id: str = CLIENT_ID
+        self._client_secret: str = CLIENT_SECRET
+        self._refresh_token: str = REFRESH_TOKEN
+        self._developer_token: str = DEVELOPER_TOKEN
+        self._tenant_id: str = TENNANT_ID
+        self._report_start_date: str = "2020-12-18"
+        self._lookback_window: int = LOOKBACK_WINDOW
+
+    def build(self) -> Dict[str, Any]:
+        config = {
+            "tenant_id": self._tenant_id,
+            "developer_token": self._developer_token,
+            "refresh_token": self._refresh_token,
+            "client_id": self._client_id,
+            "client_secret": self._client_secret,
+            "lookback_window": self._lookback_window,
+        }
+        if self._report_start_date:
```

Comment:
Can there be a situation the `self._report_start_date is None`?

Reply:
Yes, but is it possible that the `_report_start_date` is not present in this config?

Reply:
I updated config builder logic, it should be more clear now